sympy/plotting/plot.py

```diff
@@ -947,7 +1005,7 @@ def get_color_array(self):
             else:
                 return f(*variables)
         else:
-            return c*np.ones(self.nb_of_points)
+            return c*np.ones(min(self.nb_of_points_x, self.nb_of_points_y))
```

Comment:
Does this work with ParametricSurfaceSeries? I see it uses `nb_of_points_u, nb_of_points_v` instead of `x, y`

Reply:
Nope, need to fix it!

Reply:
I think that you should add some dry-run tests (without checking image output) in test_plot to see if these don’t raise some exceptions at least

Reply:
Done. Thank you for pointing that out! :)